test/units/testsuite-42.sh

```diff
@@ -65,6 +65,10 @@ test -f /run/dbus1
 systemd-run --unit=dbus2.service --wait -p StandardOutput=tty -p StandardError=tty -p Type=dbus -p BusName=systemd.test.ExecStopPost -p ExecStopPost='/bin/touch /run/dbus2' true
 test -f /run/dbus2
 
+# https://github.com/systemd/systemd/issues/19920
+systemd-run --unit=dbus3.service --wait -p StandardOutput=tty -p StandardError=tty -p Type=dbus -p ExecStopPost='/bin/touch /run/dbus3' true \
+     && { echo 'unexpected success'; exit 1; }
+
 cat >/tmp/notify1.sh <<EOF
```

Comment:
Does this properly catch the fact that PID 1 was crashing?

Reply:
If PID1 crashes, then the test freezes and hopefully CIs finish with timeout.

Reply:
And the journal should show some thing like the following:
```
Jun 15 03:18:01 H systemd[1]: Assertion 'name' failed at src/core/unit.c:3459, function unit_watch_bus_name(). Aborting.
Jun 15 03:18:01 H audit[530]: ANOM_ABEND auid=4294967295 uid=0 gid=0 ses=4294967295 pid=530 comm="systemd" exe="/usr/lib/systemd/systemd" sig=6 res=1
Jun 15 03:18:01 H systemd-coredump[531]: Due to PID 1 having crashed coredump collection will now be turned off.
Jun 15 03:18:01 H systemd-coredump[531]: 8;;man:core(5)^G[🡕]8;;^G Process 530 (systemd) of user 0 dumped core.
(snip)
```